apps/condo/domains/organization/access/Organization.js

```diff
@@ -2,13 +2,28 @@
  * Generated by `createschema organization.Organization 'country:Select:ru,en; name:Text; description?:Text; avatar?:File; meta:Json; employees:Relationship:OrganizationEmployee:CASCADE; statusTransitions:Json; defaultEmployeeRoleStatusTransitions:Json' --force`
  */
 const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
-const { checkUserIsRelatedFromOrganizationEmployee, checkIfUserIsOrganizationEmployee } = require('../utils/accessSchema')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
 const { queryOrganizationEmployeeFromRelatedOrganizationFor, queryOrganizationEmployeeFor } = require('../utils/accessSchema')
+const { Resident: ResidentServerUtils } = require('@condo/domains/resident/utils/serverSchema')
+const { get } = require('lodash')
 
-async function canReadOrganizations ({ authentication: { item: user } }) {
+async function canReadOrganizations ({ authentication: { item: user }, context }) {
     if (!user) return throwAuthenticationError()
     if (user.isAdmin || user.isSupport) return {}
     const userId = user.id
+    if (user.type === RESIDENT) {
+        const residents = await ResidentServerUtils.getAll(context, { user: { id: userId } })
+        if (residents.length === 0) {
+            return false
+        }
+        const organizations = residents.map(resident => get(resident, ['organization', 'id']))
+        if (organizations.length > 0) {
+            return {
+                id_in: organizations,
```

Comment:
When user has several residents, being served with one organization, we will have duplicated ids here.
Use uniq from lodash:
```suggestion
                id_in: uniq(organizations),
```